.env

```diff
@@ -1,4 +1,3 @@
-NODE_PATH=src
-PORT=8080
+PORT=3000
```

Comment:
8080 kills me because everything runs on that port!  I'll figure out a local override for this or something.